packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -363,7 +398,18 @@ class AnimationController extends Animation<double>
     return _animateToInternal(target, duration: duration, curve: curve);
   }
 
-  TickerFuture _animateToInternal(double target, { Duration duration, Curve curve = Curves.linear }) {
+  TickerFuture _animateToInternal(double target, { Duration duration, Curve curve = Curves.linear, AnimationBehavior animationBehavior }) {
+    final AnimationBehavior behavior = animationBehavior ?? this.animationBehavior;
+    double scale = 1.0;
+    if (SemanticsBinding.instance.disableAnimations) {
```

Comment:
also wherever you call into SemanticsBinding.instance, handle the case of it being null (which might happen if someone has code that they haven't updated to mix in the new binding).